protected/humhub/modules/admin/resources/js/humhub.admin.pending.registrations.js

```diff
@@ -0,0 +1,51 @@
+humhub.module('admin.pending.registrations', function (module, require, $) {
+    module.initOnPjaxLoad = true;
+
+    function hasChecked($checkBoxes) {
+        var result = false;
+        $checkBoxes.each(function () {
+            if ($(this).prop("checked")) {
+                result = true;
+                return false;
+            }
+        });
+        return result;
+    }
+
+    var init = function (isPjax) {
+
+        $("input").change(function () {
```

Comment:
Note, once loaded a module will be initialized on every following pjax load. In the init function you should make sure you are residing in the right view, either by checking the existence of certain dom elements (selectors) or checking the [view state](https://github.com/humhub/humhub/blob/master/static/js/humhub/humhub.ui.view.js#L167). In this case you could also consider implementing a [JsWidget](https://docs.humhub.org/docs/develop/javascript-components#widgets)

Also its never a good idea to add such handlers to all `input` elements on the page, you should be more specific about which elements you want to listen to (e.g. only within a specific root element).

For example when implementing a JsWidget on the root element of the form, you could just call `this.$.find('input')`.

Reply:
Done!